staging/src/k8s.io/apiserver/pkg/storage/cacher/delegator.go

```diff
@@ -168,8 +199,9 @@ func (c *CacheDelegator) GetList(ctx context.Context, key string, opts storage.L
 		if err != nil {
 			return err
 		}
+		opts.ResourceVersion = strconv.FormatInt(int64(listRV), 10)
```

Comment:
Overwriting opts.ResourceVersion makes sense here, given we already computed the appropriate RV.
But let's add a short comment describing why you're doing that.